CMakeLists.txt

```diff
@@ -209,9 +209,9 @@ if (PLATFORM_LINUX OR PLATFORM_ANDROID)
 endif()
 
 if (NOT SKIP_TESTS)
-  set(SKIP_TESTS FALSE)
   enable_testing()
```

Comment:
This is implied by `include(CTest)` https://cmake.org/cmake/help/latest/command/enable_testing.html
BTW, is there any reason to prefer `SKIP_TESTS` over the official `BUILD_TESTING`?